src/I18n/index.ts

```diff
@@ -44,7 +45,8 @@ export class I18n extends Formatter implements I18nContract {
     public locale: string,
     private emitter: EmitterContract,
     private logger: LoggerContract,
-    private i18nManager: I18nManagerContract
+    private i18nManager: I18nManagerContract,
+    public config: I18nConfig
```

Comment:
I also think it would be better to access the configuration via `this.i18nManager.config` instead of having to receive it as a parameter here. We keep a single configuration as the only source of truth. Makes more sense imo

Would also allow you to remove all the boilerplate you had to add for the failing tests

Reply:
Yes, that makes a lot of sense. Did not know I could access it from i18nManager, so that's my bad!
Have fixed it in the latest commit